### PR TITLE
fix chicken

### PR DESCRIPTION
**Pull request by robsideshow**
Fix some stuff with the chicken:
Use pygame.sprite.groupcollide instead of spritecollide for balls hitting chickens.  That way, killing a chicken means it won't collide anymore.  Chickens are now in a group called target_group.
Press the letter 'c' on the keyboard to add more chickens!!!
Fix the balls not colliding correctly with the rightwall.  Something about image/surface/rect/inflate/????
Change ball size to radius

### CODE REVIEW

File: ballgame.py [hunk 1]
```diff
@@ -82,6 +85,7 @@ def __init__(self, pos, imagefile, hp):
         self.rect = self.image.get_rect()
         self.rect.center = pos
         self.v = [0,0]
+        self.speed = 5
```
Comment (calypsomatic) [inline ballgame.py:88]: Why 5?  Shouldn't it be speed/2 or something?

Comment (robsideshow) [reply ballgame.py:88]: The balls go 10, so I put 5 for the chicken.  I'm OK with any speed you want.  When you say speed/2, do you mean half the ball speed?  We should probably refactor/rename our Sprite and BounceSprite classes soon.

Comment (calypsomatic) [reply ballgame.py:88]: Just that the speed is passed in for chickens, do we want a default for the balls or to pass it in as well?

Comment (robsideshow) [reply ballgame.py:88]: Args for the chicken are pos, imagefile, hp at the moment.  But I was thinking of passing in speeds for target sprites

**Pull request merged**